README.rst

```diff
@@ -14,7 +14,7 @@
 
 .. index-start-marker
 
-`Ocean <https://docs.ocean.dwavesys.com/en/stable>`_ is
+`Ocean SDK <https://docs.dwavequantum.com/en/latest/ocean/index.html>`_ is
```

Comment:
I'd leave out `index.html`, as it's not necessary, and just makes the URL longer.
```suggestion
`Ocean SDK <https://docs.dwavequantum.com/en/latest/ocean/>`_ is
```